### PR TITLE
chore: separate the feature flag

### LINKED ISSUE #5939: Increased Query Result Limit Enablement
## User Story

<p></p>

As a Product Manager, I want to be able to increase the limit of data that can be displayed to the user in the new Script Editor independently of the limit that is set for the old Data Explorer\.

<p></p>

## Acceptance Criteria

<p></p>

- A feature flag will be available, similiar to increasedCsvLimit that can be used to change the default limit in production or increase it specifically for certain org\_IDs as needed\.

<p></p>

## Notes

<p></p>

Increasing the limit for old Data Explorer has some risk because there are a lot of users that are using it\.  But it is less risky for the new Script Editor because of the smaller number of users and the "preview" conditions on which they are using it\.  So I would like to be able to raise the limit for just the new Script Editor\.

<p></p>

This could also be used as an incentive to move to the new Script Editor in the future\.

<p></p>

### PR DESCRIPTION
**Pull request by asalem1**
Closes #5939

Separates the CSV Limit flag for data explorer and the new schema browser

### CODE REVIEW

**Review by genehynson — APPROVED** [review]

**Review by wiedld — CHANGES_REQUESTED** [review]
I've been offline asking @asalem1 to make changes.

**Comment by wiedld** [comment]
I was asked to review this PR. Was having a slack convo with @asalem1 about this....and then suddenly it's approved and being merged.

I just removed from merge queue. And going to move the slack convo here for full transparency.

The issue ticket talks about having 2 feature flags:
1. increases csv limits for everywhere else
2. increase csv limits for only the new QxBuilder

This line of code changed here, is upstream called by `runQuery()`. Which is in turn used throughout the application:
Screen Shot 2022-10-03 at 12 29 58 PM


So my question were:
* is this new flag supposed to be only for the new dataExplorer?
* if yes, then don't we need the additional conditionals we use to check that it's the new QxBuilder?
   *  we do a similar thing where schemaComposition is feature flagged in the same path as the rest of the UI application:
      * https://github.com/influxdata/ui/blob/master/src/shared/components/FluxMonacoEditor.tsx#L69-L72

File: src/shared/contexts/query.tsx [hunk 1]
```diff
@@ -588,9 +589,14 @@ export const QueryProvider: FC = ({children}) => {
...
           let read = await reader.read()
 
-          const BYTE_LIMIT =
+          let BYTE_LIMIT =
             getFlagValue('increaseCsvLimit') ?? FLUX_RESPONSE_BYTES_LIMIT
 
+          if (!window.location.pathname.includes(PROJECT_NAME.toLowerCase())) {
+            BYTE_LIMIT =
+              getFlagValue('dataExplorerCsvLimit') ?? FLUX_RESPONSE_BYTES_LIMIT
+          }
```
Comment (wiedld) [inline src/shared/contexts/query.tsx:598]: Ah...doesn't this add the new feature flag `'dataExplorerCsvLimit'` to only the notebooks? And not the new Qx Builder?

**Review by wiedld — CHANGES_REQUESTED** [review]

File: src/shared/contexts/query.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (wiedld) [reply src/shared/contexts/query.tsx:598]: yep. I missed the negative. sry!

**Review by wiedld — APPROVED** [review]

**Comment by genehynson** [comment]
> I was asked to review this PR. Was having a slack convo with @asalem1 about this....and then suddenly it's approved and being merged. (the dangers of automerge being enabled...instead of waiting for approval before adding to the queue.) I just removed from merge queue. And going to move the slack convo here for full transparency.

My bad! I get notifications for all UI PRs and thought I'd help unblock this one - I was unaware of the offline context. Sorry!

**Pull request merged**